src/apps/FEMSWEEP-Cuda.cpp

```diff
@@ -0,0 +1,78 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-24, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "FEMSWEEP.hpp"
+
+#include "RAJA/RAJA.hpp"
+
+#if defined(RAJA_ENABLE_CUDA)
+
+#include "common/CudaDataUtils.hpp"
+
+#include <iostream>
+
+namespace rajaperf
+{
+namespace apps
+{
+
+template < size_t block_size >
+void FEMSWEEP::runCudaVariantImpl(VariantID vid)
+{
+  const Index_type run_reps = getRunReps();
+
+  auto res{getCudaResource()};
+
+  FEMSWEEP_DATA_SETUP;
+
+  switch ( vid ) {
+
+    case RAJA_CUDA : {
+
+      const size_t grid_size = RAJA_DIVIDE_CEILING_INT(na*ng, block_size);
+      constexpr bool async = true;
+
+      using launch_policy =
+          RAJA::LaunchPolicy<RAJA::cuda_launch_t<async, block_size>>;
+
+      using outer_x =
+          RAJA::LoopPolicy<RAJA::cuda_global_thread_x>;
```

Comment:
I think we can use the policies that take the fixed blocksize like `RAJA::cuda_global_size_x_direct<block_size>`.